app/templates/reading_list.html

```diff
@@ -0,0 +1,65 @@
+{% extends "base_root.html" %}
+
+{% block navlinks %}
+  {{ super() }}
+{% endblock %}
+
+{% block main_body %}
+
+<div class="container">
+<div id="readinglist">
+    <div class="col-lg-10 ml-auto mr-auto">
+    	<h2 class="text-left title text-black">Your Book List</h2>
+        <div class="thumbnail">
+        	<div class ="row">
+        		<div class ="col-2">
+        			<div class="img-container">
```

Comment:
e.g. here, i do not know where the end-tag for the `<div class="img-container">` is, because of wrong indentation